src/main/java/org/folio/circulation/domain/LoanRepository.java

```diff
@@ -232,22 +233,15 @@ private static void keepLatestItemStatus(Item item, JsonObject storageLoan) {
     storageLoan.put(ITEM_STATUS, item.getStatus().getValue());
   }
 
-  private static void updateLastLoanPolicyUsedId(JsonObject storageLoan,
-                                                 LoanPolicy loanPolicy) {
+  private static void updatePolicy(JsonObject storageLoan,
+                                   CirculationPolicy policy,
+                                   String policyName) {
 
-    if(nonNull(loanPolicy) && loanPolicy.getId() != null) {
-      storageLoan.put("loanPolicyId", loanPolicy.getId());
+    if (nonNull(policy) && policy.getId() != null) {
+      storageLoan.put(policyName, policy.getId());
     }
   }
 
-  private static void removeFeesAndFinesProperties(JsonObject storageLoan) {
-    storageLoan.remove(LoanProperties.FEESANDFINES);
-  }
-
-  private static void removeChangeMetadata(JsonObject storageLoan) {
-    storageLoan.remove("metadata");
-  }
```

Comment:
Why has this method been removed?

Reply:
Used removeProperty method instead of removeChangeMetadata